lib/matplotlib/figure.py

```diff
@@ -876,18 +876,24 @@ def figimage(self, X, xo=0, yo=0, alpha=None, norm=None, cmap=None,
         return im
 
     def set_size_inches(self, w, h=None, forward=True):
-        """Set the figure size in inches.
+        """
+        Set the figure size in inches.
 
         Call signatures::
 
              fig.set_size_inches(w, h)  # OR
              fig.set_size_inches((w, h))
 
-        optional kwarg *forward=True* will cause the canvas size to be
-        automatically updated; e.g., you can resize the figure window
-        from the shell
-
-        ACCEPTS: a (w, h) tuple with w, h in inches
+        Parameters
+        ----------
+        w : tuple of floats or float
+            Width and height in inches (if height not specified as a separate
+            argument) or width.
+        h : float
```

Comment:
```suggestion
        h : float, optional
```

Specifying the default here does not make lots of sense due to the weird interaction with `w`.